src/test/java/com/fasterxml/jackson/databind/type/TestTypeFactory.java

```diff
@@ -727,18 +760,20 @@ public void testMoreSpecificType()
     }
 
     // [databind#489]
+    @Test
     public void testCacheClearing()
     {
         TypeFactory tf = TypeFactory.defaultInstance().withModifier(null);
         assertEquals(0, tf._typeCache.size());
         tf.constructType(getClass());
         // 19-Oct-2015, tatu: This is pretty fragile but
-        assertEquals(6, tf._typeCache.size());
+        assertTrue(tf._typeCache.size() > 0);
```

Comment:
Because we changed test runner to JUnit5, the `tf._typeCache.size()` returns `1` and break the test. So changed this test to focus more on the `clearCache()` method.

Reply:
Ok, sounds reasonable. And probably is actually more accurate too.